extra/1-radio-stations.js

```diff
@@ -13,6 +15,14 @@
  * - Should return this array to use in other functions
  */
 
+function getAllFrequencies() {
+	let frequencies = [];
+	for (let i = 87; i < 109; i++) {
+		frequencies[i - 87] = i;
+	}
+	return frequencies;
+}
```

Comment:
Hi Anza, just a thought : could you consider using .push() here instead of frequencies[i - 87] = i ? Otherwise your code is very good. Thank you.